cms/templates/js/course-outline.underscore

```diff
@@ -122,9 +122,15 @@ if (is_proctored_exam) {
                     </span>
                 <% } %>
             </h3>
-
         <div class="<%- xblockType %>-header-actions">
             <ul class="actions-list">
+                <% if (xblockInfo.isSequential() && xblockInfo.get('proctoring_provider') === 'proctortrack') { %>
```

Comment:
This can be conditional based on `xblockInfo.get('proctoring_exam_configuration_link')` rather than on the particular backend name.